<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_select_target_dir">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border: none;
text-decoration: underline;
font-weight: bold;
color: rgb(255, 0, 0);
font-size: 20px;
}

QPushButton:hover {
color: blue;
}</string>
   </property>
   <property name="text">
    <string>select/target/dir</string>
   </property>
  </widget>
  <widget class="QFrame" name="drag_n_drop_area">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
  background-color: rgb(204, 204, 204);
  border-radius: 12px 0;
}

QFrame:hover {
  opacity: 0.9;
}</string>
   </property>
   <widget class="QLabel" name="drag_n_drop_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-size: 21px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Drag &amp;amp; Drop Files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="files_view">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView 
{
background-color: transparent;
border: none;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textedit_targetdir">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
  border: none;
  background: transparent;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:14pt;&quot;&gt;Target Directory:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(8, 180, 42);
color: white;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_target_dir">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
